docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: '3.9'
+
+services:
+  database:

```

Comment:
I think you missed container for starting up the aac itself, cause the docker-compose should contain ready to use aac environment

Reply:
In fact, I'm just running Docker, to have a database, it's functional, but in the future we can add scripts to run the backend, such as migrations, seeds, etc.